package/carbonio-files.hcl

```diff
@@ -61,6 +61,11 @@ services {
             destination_name   = "carbonio-docs-connector"
             local_bind_address = "127.78.0.2"
             local_bind_port    = 20005
+          },
+          {
+            destination_name   = "carbonio-message-broker"
+            local_bind_address = "127.78.0.2"
+            local_bind_port    = 20006
```

Comment:
Same thing as the comment in `config.properties`. Let's think a solution together